<launch>
<arg name="uav_id" default="0" />
<arg name="t_b_file" default="/home/ubuntu/catkin_ws/transform_data.yaml" />
<arg name="k_imu" default="/home/ubuntu/catkin_ws/uav_$(arg uav_id)-imu.yaml" />
<arg name="k_cam_imu" default="/home/ubuntu/catkin_ws/uav_$(arg uav_id)-camchain-imucam.yaml" />
<node pkg="sensor_config" name="publish_params_tp_tf_$(arg uav_id)" type="publish_params_tp_tf" output="screen" ns="uav_$(arg uav_id)" args="--uav_id=$(arg uav_id) --t_b_file=$(arg t_b_file) --k_imu=$(arg k_imu) --k_cam_imu=$(arg k_cam_imu)"/>
</launch>

